docs/modules/ROOT/pages/local-mode.adoc

```diff
@@ -23,3 +23,27 @@ compile the catalog for the selected cluster.
 --
 poetry run commodore catalog compile <cluster-id>
 --
+
+To test a component which is new to a cluster in local mode, the component has to be downloaded and symlinked into the directory structure of a previous `commodore catalog compile`.
+
+[source,bash]
+--
+COMPONENT_NAME=
+COMPONENT_GIT_URL=
+
+git clone ${COMPONENT_GIT_URL} ./dependencies/${COMPONENT_NAME}
+
+ln -svf ../../../dependencies/${COMPONENT_NAME}/class/defaults.yml ./inventory/classes/defaults/${COMPONENT_NAME}.yml
+ln -svf ../../../dependencies/${COMPONENT_NAME}/class/${COMPONENT_NAME}.yml ./inventory/classes/components/${COMPONENT_NAME}.yml
+
+ln -svf ../dependencies/${COMPONENT_NAME} ./vendor/${COMPONENT_NAME}
```

Comment:
Instead of trying to do jsonnet-bundler's work by hand, simply update the `jsonnetfile.json` and run `jb install`:
```suggestion
jq --arg component_name $COMPONENT_NAME -r '.dependencies += [{"source":{"local":{"directory":"dependencies/\($component_name)"}}}]' jsonnetfile.json > jsonnetfile.json.new && mv jsonnetfile.json.new jsonnetfile.json
jb install
```